style.css

```diff
@@ -0,0 +1,784 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1400px;
+  padding: 0 24px;
+}
+
+header {
+  width: 100%;
+}
+
+.nav-top {
+  background-color: black;
+}
+
+.navbar-top {
+  height: 35px;
+  display: flex;
+  align-items: center;
+  justify-content: end;
+}
+
+.navbar-top-items {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-top-item {
+  display: flex;
+  align-items: center;
+}
+
+.navbar-top-item-link {
+  align-self: center;
+  margin-left: 15px;
+  color: #fff;
+}
+
+.nav-bottom-container {
+  background-color: #fff;
+}
+
+.nav-bottom {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo-img {
+  width: 100px;
+  height: auto;
+}
+
+.logo-link {
+  order: -1;
+}
+
+.mobile-menu-button-link {
+  display: none;
+}
+
+.navbar-bottom-items {
+  display: flex;
+}
+
+.navbar-bottom-item-link {
+  color: black;
+  margin-left: 60px;
```

Comment:
- [ ] Kindly ensure to follow the color scheme provided in the Behance design. The nav item links should not be black. kindly use `#272a31`
![image](https://user-images.githubusercontent.com/46839548/163278497-0470a2a6-f39d-4d19-9530-8eb0a146aba3.png)

Reply:
Hello codepantha! Thank you for the requested change. I kindly changed the color for this element as per the design template.